dlc/src/main/scala/org/bitcoins/dlc/DLCClient.scala

```diff
@@ -238,6 +239,38 @@ case class DLCClient(
     sigsMapF.map(FundingSignatures.apply)
   }
 
+  def verifyRemoteFundingSigs(remoteSigs: FundingSignatures): Boolean = {
+    val remoteTweak = if (isInitiator) {
+      fundingUtxos.length
+    } else {
+      0
+    }
+
+    val psbt = PSBT.fromUnsignedTx(createUnsignedFundingTransaction)
+
+    remoteSigs.zipWithIndex
+      .foldLeft(true) {
+        case (ret, ((outPoint, sigs), index)) =>
+          if (ret) {
+            require(psbt.transaction.inputs(index).previousOutput == outPoint,
+                    "Adding signature for incorrect input")
+
+            val idx = index + remoteTweak
+
+            // TODO: add funding witness and redeem scripts
```

Comment:
is there maybe an issue filed for this TODO and the one below?

Reply:
https://github.com/bitcoin-s/bitcoin-s/issues/1344